lib/commonmarker/config.rb

```diff
@@ -7,23 +7,28 @@ module Config
     OPTS = {
       parse: {
         DEFAULT: 0,
+        SOURCEPOS: (1 << 1),
```

Comment:
As far as I can tell, we check for `SOURCEPOS` in `inlines.c`, thereby allowing someone implementing their own renderer to also add sourcepos info to their output.